python/ray/autoscaler/gcp/tpu.yaml

```
@@ -34,40 +14,26 @@ available_node_types:
             runtimeVersion: v2-alpha
             schedulingConfig:
                 # Set to false to use non-preemptible TPUs
+                preemptible: false
+    ray_tpu:
+        min_workers: 1
+        resources: {"TPU": 1}  # use TPU custom resource in your code
+        node_config:
+            acceleratorType: v2-8
+            runtimeVersion: v2-alpha
+            schedulingConfig:
                 preemptible: true
 
 provider:
     type: gcp
     region: us-central1
-    availability_zone: us-central1-b
-    project_id: null  # replace with your GCP project id
+    availability_zone: us-central1-f
```

```suggestion
    availability_zone: us-central1-b
```

I couldn't get it to work with `us-central1-f`.  The docs say "Note: Zone us-central1-f is available only to [TPU Research Cloud (TRC) participants](https://sites.research.google/trc)."  So perhaps `b` will be better for more users.

---

(doc link) https://cloud.google.com/tpu/docs/regions-zones#overview